<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ката</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0" columnstretch="0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>40</number>
      </property>
      <property name="topMargin">
       <number>25</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>7</width>
          <height>100</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0">
        <property name="spacing">
         <number>40</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font/>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{

background-color: #e2e2e2;
border:1px solid #afafaf;
}
QPushButton:hover{
background-color: #E0F2FF;
border-color: #80C8FF;
}
QPushButton:pressed{
background-color: #C2E4FF;
border-color: #80C8FF;
}</string>
          </property>
          <property name="text">
           <string>Начать соревнования</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="select_not_end_comp">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{

background-color: #e2e2e2;
border:1px solid #afafaf;
}
QPushButton:hover{
background-color: #E0F2FF;
border-color: #80C8FF;
}
QPushButton:pressed{
background-color: #C2E4FF;
border-color: #80C8FF;
}</string>
          </property>
          <property name="text">
           <string>Продолжить соревнование</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ended_compettition">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font/>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{

background-color: #e2e2e2;
border:1px solid #afafaf;
}
QPushButton:hover{
background-color: #E0F2FF;
border-color: #80C8FF;
}
QPushButton:pressed{
background-color: #C2E4FF;
border-color: #80C8FF;
}</string>
          </property>
          <property name="text">
           <string>Проведённые соревнования и отчёты</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="categories_mode">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font/>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{

background-color: #e2e2e2;
border:1px solid #afafaf;
}
QPushButton:hover{
background-color: #E0F2FF;
border-color: #80C8FF;
}
QPushButton:pressed{
background-color: #C2E4FF;
border-color: #80C8FF;
}</string>
          </property>
          <property name="text">
           <string>Режим дополнительного татами</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_not_ended_comp">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{

background-color: #e2e2e2;
border:1px solid #afafaf;
}
QPushButton:hover{
background-color: #E0F2FF;
border-color: #80C8FF;
}
QPushButton:pressed{
background-color: #C2E4FF;
border-color: #80C8FF;
}</string>
          </property>
          <property name="text">
           <string>Удалить незаконченое мероприятие</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="go_to_options">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>55</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{

background-color: #e2e2e2;
border:1px solid #afafaf;
}
QPushButton:hover{
background-color: #E0F2FF;
border-color: #80C8FF;
}
QPushButton:pressed{
background-color: #C2E4FF;
border-color: #80C8FF;
}</string>
          </property>
          <property name="text">
           <string>Настройки</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="open_instructions">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>55</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{

background-color: #e2e2e2;
border:1px solid #afafaf;
}
QPushButton:hover{
background-color: #E0F2FF;
border-color: #80C8FF;
}
QPushButton:pressed{
background-color: #C2E4FF;
border-color: #80C8FF;
}</string>
          </property>
          <property name="text">
           <string>Открыть инструкцию</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>240</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" rowspan="3">
       <widget class="QLabel" name="pict">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>430</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
